DogeNet.WebApi/Startup.cs

```diff
@@ -103,6 +105,7 @@ public void ConfigureServices(IServiceCollection services)
                     );
             });
             services.AddSignalR();
+            services.AddSingleton<IDictionary<string, UserConnection>>(opts => new Dictionary<string, UserConnection>());
```

Comment:
What is this for?

Reply:
@oleh-leobit this is signalR connections dictionary, is used here
![image](https://user-images.githubusercontent.com/77879337/170702589-a1afaa67-bf8f-4636-b64c-b16741a2e1c7.png)